test_sets/External_UCS-related_params.txt

```diff
@@ -0,0 +1,22 @@
+./test_scripts/Polices/External_UCS/001_ATF_P_TC_PreloadedPT_without_external_consent_status_groups_struct.lua Check_that_SDL_starts_successfully_in_case_external_consent_status_groups_struct_is_omitted_in_PreloadedPT
+./test_scripts/Polices/External_UCS/002_ATF_N_TC_PreloadedPT_with_external_consent_status_groups_struct.lua Check_that_SDL_shut_down_in_case_external_consent_status_groups_struct_is_defined_in_PreloadedPT
+./test_scripts/Polices/External_UCS/003_ATF_P_TC_PreloadedPT_without_disallowed_by_external_consent_entities_off_struct.lua Check_that_SDL_starts_successfully_in_case_disallowed_by_external_consent_entities_off_struct_is_omitted_in_PreloadedPT
+./test_scripts/Polices/External_UCS/004_ATF_P_TC_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct.lua Check_that_SDL_stores_disallowed_by_external_consent_entities_off_struct_in_LPT_in_case_it_is_defined_in_PreloadedPT
+./test_scripts/Polices/External_UCS/005_ATF_P_TC_PreloadedPT_without_disallowed_by_external_consent_entities_on_struct.lua Check_that_SDL_starts_successfully_in_case_disallowed_by_external_consent_entities_on_struct_is_omitted_in_PreloadedPT
```

Comment:
@dboltovskyi, as I understand this script is based on requirement:"[Policies] External UCS: PreloadedPT without "disallowed_by_external_consent_entities_on" struct"

This requirement should be checked in 3 cases:
first SDL run
after PreloadedPT update
after FactoryDefaults